manifests/init.pp

```diff
@@ -9,22 +11,19 @@
 # @param install_dir [String] Directory to install module into (Default: "/opt/puppetlabs/pe_databases")
 # @param scripts_dir [String] Directory to install scripts into (Default: "${install_dir}/scripts")
 class pe_databases (
-  # Manage the inclusion of the pg_repack class
   Boolean $manage_database_maintenance = true,
-  # Manage the state of the maintenance tasks, i.e. systemd services and timers
-  Boolean $disable_maintenance         = lookup('pe_databases::disable_maintenance', {'default_value' => false}),
+  Boolean $disable_maintenance         = false,
```

Comment:
Most of these changes are white spacing, and i agree, however this one changes what we want the default to be

Reply:
@MartyEwings You are referring to the explicit lookup? The lookup matches the automatic lookup behavior, so the lookup can be removed, and the default is still the same value.